electron/helpers/utils.js

```diff
@@ -10,8 +11,45 @@ export {
   v4 as generateKey
 } from 'uuid'
 
+const harmfulSwitches = [
```

Comment:
@xyloflake Please create `data.js` in this folder and move it there. Them import it here.

Reply:
Okay.